WooCommerce/Classes/POS/Presentation/ItemCardView.swift

```diff
@@ -28,20 +31,15 @@ struct ItemCardView: View {
                            height: Constants.productImageWidth * scale)
                     .foregroundColor(.gray)
             }
-            VStack {
+            VStack(alignment: .leading) {
                 Text(item.name)
                     .foregroundStyle(Color.primaryBackground)
-                Text(item.price)
+                Text(commaSeparatedItemCategories)
                     .foregroundStyle(Color.primaryBackground)
-                HStack(spacing: 8) {
-                    Spacer()
-                    Button(action: {
-                        onItemCardTapped?()
-                    }, label: { })
-                    .buttonStyle(POSPlusButtonStyle())
```

Comment:
Agree, done in https://github.com/woocommerce/woocommerce-ios/pull/13069/commits/5978028697554e58d3da485d666e1a9b5afb2aab. I've also deleted the `ItemGridView` in https://github.com/woocommerce/woocommerce-ios/pull/13069/commits/a8e1b5b473848c72b395bd972da472a1a88efb40 since might be easier to redo it from latest designs if we happen to need it.